### PR TITLE
[GH-15898] Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 668

### PR DESCRIPTION
**Pull request by xaylax**
#### Summary
Removes the line: changeCss('.app__body .post-collapse_*show-more-button:hover', 'color:${theme.centerChannelBg}'); from `utils/utils.jsx` and replaces it with a new color variable under the ` .post-collapse__show-more-button` section in `sass/components/_post.scss `

#### Ticket Link
Fixes: https://github.com/mattermost/mattermost-server/issues/15898

### CODE REVIEW

**Comment by xaylax** [comment]
hi @devinbinnie! Thanks so much for the comment, that totally makes sense- I've fixed the file and the changes are now reflected in my pull :)

**Review by devinbinnie — APPROVED** [review]
Looks good now, thanks @xaylax!

**Comment by deanwhillier** [comment]
QA on this PR will be batched when the campaign branch is merged to master.

**Review by asaadmahmood — APPROVED** [review]

File: utils/utils.jsx [hunk 1]
```diff
@@ -665,7 +665,6 @@ export function applyTheme(theme) {
             `background-color:${theme.centerChannelBg}`,
         );
 
-        changeCss('.app__body .post-collapse__show-more-button:hover', `color:${theme.centerChannelBg}`);
```
[the remark this answers is not in the document]
Comment (deanwhillier) [reply utils/utils.jsx:668]: @asaadmahmood, looks like the PR is now targeting the hover state, good to merge?

**Pull request merged**